### ISSUE TITLE
Error: 500:Error: Expected integer between 1 and 16383 for width but received NaN of type number

### ISSUE THREAD

**Issue by jacobgc** [jacobgc/place.cat#1]
https://sentry.io/jacobgc/placecat/issues/278203049/

```
Error: 500:Error: Expected integer between 1 and 16383 for width but received NaN of type number
  File "/root/PROD/place.cat/app.js", line 38, in null.<anonymous>
    Raven.captureException('500:' + err);
...
(11 additional frame(s) were not displayed)

Error: 500:Error: Expected integer between 1 and 16383 for width but received NaN of type number
```

**Comment by jacobgc** [comment]
Problem solved in #661da52793a55eaf422705355849301c43681687

**Comment by jacobgc** [comment]
solved in https://github.com/jacobgc/place.cat/commit/661da52793a55eaf422705355849301c43681687
